test/SourceKit/CodeComplete/rdar71566576.swift

```diff
@@ -0,0 +1,3 @@
+// RUN: %swift-ide-test -code-completion -code-completion-token=COMPLETE -source-filename %s
+
+if let item = ["a"].first(where: { #^COMPLETE^# }) {}
```

Comment:
Does it need to check actual completions produced here or was it a crasher?

Reply:
It was a crasher. Do you think it would be worthwhile still checking that we actually produce some completions?

Reply:
It's up to you if the completions are good :) If it's just to make sure that compiler doesn't crash I'd suggest we move this to `validation-test/IDE/crashers_fixed`

Reply:
Thanks for the pointer. I didn’t know about `validation-test/IDE/crashers_fixed`. Already applied the new knowledge to https://github.com/apple/swift/pull/36523 😉

Reply:
No problem!